aspnetcore/blazor/components/prerender.md

```diff
@@ -125,6 +130,12 @@ For components embedded into a page or view of a Razor Pages or MVC app, you mus
 </body>
 ```
 
+## Interactive routing and prerendering
+
+Internal navigation for interactive routing doesn't involve requesting new page content from the server. Therefore, prerendering doesn't occur for internal page requests.
+
+The <xref:Microsoft.AspNetCore.Components.PersistentComponentState> service only works on the initial page load and not across enhanced page navigation events. If the app performs a full (non-enhanced) navigation to a page utilizing persistent component state, the persisted state is made available for the app to use when it becomes interactive. But if an interactive circuit has already been established and an enhanced navigation is performed to a page that renders persisted component state, that state isn't made available in the existing circuit. The <xref:Microsoft.AspNetCore.Components.PersistentComponentState> service isn't aware of enhanced navigation, and there's no mechanism to deliver state updates to components that are already running.
```

Comment:
Since this is about enhanced navigation, which is a type of static (non-interactive) navigation, should this paragraph go somewhere else? I'm also wondering if we should cross-reference any docs that discuss enhanced navigation in case this is the first time the reader is seeing this term.

Reply:
(referring to that last paragraph, I'm not sure why GitHub keeps including the previous paragraph in the review comment)

Reply:
I don't think so because this is the first place that the service is described and it only pertains to the subject of the article, which is prerendering.

Enhanced nav already cross-links to this, so I think I just need to cross-link this back out to the enhanced nav coverage in the *Routing* article.

After this PR, I'll keep an 👂 open as usual for reader feedback.